python/dgl/heterograph.py

```diff
@@ -2614,6 +2614,76 @@ def is_homogeneous(self):
         """
         return len(self.ntypes) == 1 and len(self.etypes) == 1
 
+    def is_bidirected(self, etype=None, eweight=None):
+        """Return whether the graph is a bidirected graph.
+
```

Comment:
```
...

A graph is bidirected if for any edge (u, v) in G with weight w, there exists an edge
(v, u) in G with the same weight.
```

Is this how you implement it? What if the graph has parallel edges?

Reply:
Yes. I've updated the doc and implementation. It now also handles parallel edges with different edge weights.